SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-intro.md

```diff
@@ -1,21 +1,24 @@
-# Example Project: Multiple deployments for session-based games
+# Multiple deployments for session-based games
+> **This tutorial uses the Example Project from the GDK's [Set up guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro)**</br>
+
 
 In this tutorial, you will use the [Deployment Manager](https://github.com/spatialos/deployment-manager) to launch multiple deployments of the [Example Project]({{urlRoot}}/content/get-started/example-project/exampleproject-intro) in the cloud.</br>
 
+Multiple deployments of your game are useful for time-limited match-based games where you have multiple game sessions with a defined number of players, such as battle royales.  The tutorial guides you through cloning the SpatialOS Deployment Manager repository and configuring the Example Project to work with the Deployment Manager, before launching your deployments via the Deployment Manager.
 
-Multiple deployments of your game are useful for time-limited match-based games where you have multiple game sessions with a defined number of players, such as battle royales.</br>
+**NOTES:** 
 
-**NOTE:** You can only use the Deployment Manager in the cloud; it isn't available for local deployments.
+* You can only use the Deployment Manager in the cloud; it isn't available for local deployments.
+* Before starting this tutorial you need to have completed the [Example Project set up guide.]({{urlRoot}}/content/get-started/example-project/exampleproject-intro) 
```

Comment:
```suggestion
* Before starting this tutorial you need to complete the [Example Project set up guide.]({{urlRoot}}/content/get-started/example-project/exampleproject-intro) 
```

(this is Eddie talking in my head, saying "_need to have completed_ would be hard to translate!")